src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,140 @@
+package Duke;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Duke {
+    //Constants
+    private static final String MESSAGE_BOUNDARY = "____________________________________________________________";
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    //class variables
+    private List<Task> myTasks;
+
+    //Constructor for class Duke.Duke
+    public Duke() {
+        this.myTasks = new ArrayList<>();
+    }
+
+    //Default response template to user
+    private void dukeResponse(String sampleText) {
+        System.out.println(MESSAGE_BOUNDARY);
+        System.out.println(sampleText);
+        System.out.println(MESSAGE_BOUNDARY);
+
+    }
+    //Print out full task list
+    private void dukeResponse(String sampleText , List<Task> listOfTasks) {
```

Comment:
Maybe the function names can be a bit more descriptive like for this one what prints the full task list it could be printTaskList or something?